### ISSUE TITLE
Issues with Video classification task.. Effect of multidimentional input to the network 

### ISSUE THREAD

**Issue by ashwinnair14** [BVLC/caffe#4213]
Video classification task in caffe .

I have a neural network I have to train with videos(group of images). I can choose to change the shape of the input to the network from several options.

In all cases I assume that the network architecture (arrangement and number of layers) & learning parameters (LR/decay/Regularization/etc) to be constant.

For example I could choose to give my input to the network as one of the following.

1) batch_size x (no_of_imgs*no_of_channels) x height x width {3 dimensional input}

2) batch_size x no_of_imgs x no_of_channels x height x width {4 dimensional input}

3) batch_size x no_of_channels x no_of_imgs x height x width {4 dimensional input}

How would the input shape influence the accuracy of the network? 

I had tried the 1) approach with only an accuracy of 5%. But when I use a single frame approach I get a accuracy of 60%. 
Is there a bug when I give a multi frame input?

**Comment by LisaAnne** [comment]
Hi!  Please ask questions like this on caffe-users: https://groups.google.com/forum/#!forum/caffe-users.  In response to your question, some layers (e.g., inner product, softmax) can actually take 5-d blobs as input, so that might be a good option for you.

**Comment by seanbell** [comment]
From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:
> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.
